docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+```
+
+::: warning PATCH VS PUT
+* **PATCH** partially updates the item with the provided data, any missing data is ignored
+* **PUT** fully updates the item and any missing data will fallback to its default value
+:::
+
+
+#### Body
+A single item to be updated. Field keys must match the collection's column names
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 200 OK                   | `data`: The updated item, including default fields added by Directus  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
+
+#### Examples
+
+* Return the project item with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/items/projects/1
+  ```
+
+
+
+
+### Update Items
+Update multiple items within a given collection
+@TODO WHY DOES THIS NOT USE A CSV OF IDS WITH UPDATE ITEM?
+
+```http
+PATCH /items/[collection-name]
+PUT /items/[collection-name]
+```
+
+::: warning PATCH VS PUT
+* **PATCH** partially updates the item with the provided data, any missing data is ignored
+* **PUT** fully updates the item and any missing data will fallback to its default value
+:::
+
+#### Body
+An array of items to be updated. Field keys must match the collection's column names. Alternatively, you can use the following syntax to overwrite the values of multiple items at once (Batch Update).
+
+```json
+{
+  keys: [2, 15, 31],
+  values:  {
+    name: value
+  }
+}
+```
+
+::: danger WARNING
+Batch Update can quickly overwrite large amounts of data. Please be careful when implementing this request.
+:::
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 200 OK                   | `data`: The updated item, including default fields added by Directus  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
+
+
+
+
+### Delete Item
+Deletes an item from a specific collection
+
+```http
+DELETE /items/[collection-name]/[primary-key-value]
+```
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 204 No Content           | Record was successfully deleted        |
+| 404 Not Found            | `message`: Item doesn't exist within collection    |
+
+
+
+
+### Delete Items
+Deletes multiple items from a specific collection
+
+```http
+DELETE /items/[collection-name]/
+```
+
+#### Body
```

Comment:
This should be `DELETE /items/[collection-name]/[pk],[pk]` ref https://github.com/directus/api/issues/91